content/pl/docs/reference/glossary/kube-apiserver.md

```diff
@@ -4,21 +4,20 @@ id: kube-apiserver
 date: 2018-04-12
 full_link: /docs/concepts/overview/components/#kube-apiserver
 short_description: >
-  Składnik warstwy sterowania udostępniający API Kubernetes. 
+  Składnik warstwy sterowania udostępniający API Kubernetesa.
 
 aka:
 - kube-apiserver
 tags:
 - architecture
 - fundamental
 ---
- Składnik *master* udostępniający API Kubernetes. Służy jako *front-end* dla warstwy sterowania Kubernetes.
- Serwer API jest składnikiem
-{{< glossary_tooltip text="warstwy sterowania" term_id="control-plane" >}} Kubernetes, który udostępnia API.
+Serwer API jest składnikiem
+{{< glossary_tooltip text="warstwy sterowania" term_id="control-plane" >}} Kubernetesa, który udostępnia API.
```

Comment:
Trochę masło maślane, ale nie mam pomysłu jak inaczej napisać ;-)

Reply:
Taki maślany jest oryginał...

Reply:
Powinno być ", który wystawia API na zewnątrz", ale to zbyt potoczne.